src/index.ts

```diff
@@ -69,11 +69,35 @@ async function main() {
   const body = bodyHeader + "\n\n" + results + "\n\n" + bodyFooter
 
   if (issue_number != null) {
-    await octokit.rest.issues.createComment({
-      ...repo,
-      issue_number,
-      body,
-    })
+    let comment_id: number | undefined
+    if (bodyHeader.length > 0) {
+      const comments = await octokit.rest.issues.listComments({
+        ...repo,
+        issue_number,
+        per_page: 100,
+      })
+      if (comments.status === 200) {
+        for (const comment of comments.data) {
+          if (comment.body?.includes(bodyHeader)) {
+            comment_id = comment.id
+            break
+          }
+        }
+      }
+    }
+    if (comment_id !== undefined) {
+      await octokit.rest.issues.updateComment({
```

Comment:
I think update or create should be controlled by action input.

https://github.com/int128/comment-action#inputs

Reply:
https://github.com/Quramy/simple-coverage-diff-action/pull/119/commits/aa51711dcae86a3c85337d0eb72e9d7c90a29701

This fix has differences if `update-if-exists` is `append` between `int128/comment-action`:

| update-if-exists | Condition | This pull request | int128/comment-action |
| :--------------: | :-------: | :---------------: | :-------------------: |
| Default | - | Create new comment | Create new comment |
| replace | Found | Update old comment body | Update old comment body |
| replace | Not Found | Create a comment | Create a comment |
| append | Found | **Create new comment** | **Append text to old comment body** |
| append | Not Found | Create a comment | Create a comment |